content/posts/cjd-casc.md

```diff
@@ -0,0 +1,548 @@
+---
+author:
+  name: "Matt Elgin"
+title: "Self-Updating Jenkins: GitOps for Jenkins Configuration"
+date: 2019-07-01T17:00:00-04:00
+showDate: true
+draft: true
+tags: ["jenkins","cloudbees jenkins distribution","gitops","kubernetes","kaniko","docker","cert-manager"]
+---
+
+In this blog post, we'll walk through creating a self-updating instance of [CloudBees Jenkins Distribution](https://www.cloudbees.com/products/cloudbees-jenkins-distribution) fully configured-as-code stored in a Git repository.
```

Comment:
perhaps add 'the' - so it reads "creating a self-updating instance of the [CloudBees Jenkins Distribution]"